hack/patches/0001-Patch-to-fix-generate-groups.sh.patch

```diff
@@ -0,0 +1,40 @@
+From 682f06406c73726775c973f9abc72a0b80a97bad Mon Sep 17 00:00:00 2001
+From: Jon Donovan <jondonovan@google.com>
+Date: Thu, 18 Jun 2020 23:12:55 -0700
+Subject: [PATCH] Patch to fix generate-groups.sh
+
+---
+ vendor/k8s.io/code-generator/generate-groups.sh          | 2 +-
+ vendor/k8s.io/code-generator/generate-internal-groups.sh | 3 ++-
+ 2 files changed, 3 insertions(+), 2 deletions(-)
+
+diff --git a/vendor/k8s.io/code-generator/generate-groups.sh b/vendor/k8s.io/code-generator/generate-groups.sh
+index d82002d..e2b694a 100755
+--- a/vendor/k8s.io/code-generator/generate-groups.sh
++++ b/vendor/k8s.io/code-generator/generate-groups.sh
+@@ -50,7 +50,7 @@ shift 4
+   # To support running this script from anywhere, we have to first cd into this directory
+   # so we can install the tools.
+   cd "$(dirname "${0}")"
+-  go install ./cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
```

Comment:
according to https://github.com/golang/go/issues/27653 we should be using `go run ...`

Reply:
I think this is necessary because we expect the built binaries to be consumed by other scripts

Reply:
Ah, suppose I could change /those/ though.

Reply:
Resolving, I think this is a pretty minimal diff to get what we want, lmk if you'd prefer I change.